our-team.php

```diff
@@ -3,180 +3,185 @@
-                  </div>
-                  <div class="cell-md-6">
-                    <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
-                      <h6 class="h6-with-small"><a href="#"> Sebastian King</a><span class="small text-silver-chalice">Webmaster</span></h6>
-		      <p>I am the Webmaster for UNT Robotics. I'm in charge of the website that you see in front of you. I also integrate our technology stack with other organisations and maintain the online equipment checkout system.</p>
-                      <p>I'm almost finished studying Computer Science and Electrical Engineering, so robotics is the natural choice for all of my talents. I love to be hands on with technology, and take apart and break anything that I can. For me there's nothing quite like finally seeing a project successfully working out in the real-world.
-</p>
-                      <ul class="list-inline-lg">
-			<li><a href="https://www.linkedin.com/in/sebastian-king" class="icon icon-sm text-primary fa-linkedin"></a></li>
-                        <li><a href="https://www.twitter.com/@thekingseb" class="icon icon-sm text-primary fa-twitter"></a></li>
-                        <li><a href="https://www.github.com/sebastian-king" class="icon icon-sm text-primary fa-github"></a></li>
-			<li><a href="mailto:webmaster@untrobotics.com" class="icon icon-sm text-primary fa-envelope"></a></li>
-                      </ul>
-                    </div>
-                  </div>
+              <div class="cell-md-6">
+                <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
+                  <h6 class="h6-with-small"><a href="#"> Sebastian King</a><span class="small text-silver-chalice">Webmaster</span></h6>
+                  <p>I am the Webmaster for UNT Robotics. I'm in charge of the website that you see in front of you. I also integrate our technology stack with other organisations and maintain the online equipment checkout system.</p>
+                  <p>I'm almost finished studying Computer Science and Electrical Engineering, so robotics is the natural choice for all of my talents. I love to be hands on with technology, and take apart and break anything that I can. For me there's nothing quite like finally seeing a project successfully working out in the real-world.
+                  </p>
+                  <ul class="list-inline-lg">
+                    <li><a href="https://www.linkedin.com/in/sebastian-king" class="icon icon-sm text-primary fa-linkedin"></a></li>
+                    <li><a href="https://www.twitter.com/@thekingseb" class="icon icon-sm text-primary fa-twitter"></a></li>
+                    <li><a href="https://www.github.com/sebastian-king" class="icon icon-sm text-primary fa-github"></a></li>
+                    <li><a href="mailto:webmaster@untrobotics.com" class="icon icon-sm text-primary fa-envelope"></a></li>

```

Comment:
Do we need this? Should we have this for everyone?

Reply:
Yes we absolutely should have those for everyone.

Reply:
To be clear, Emails

Reply:
Yes, everyone should have an @untrobotics.com email under their name.